js/utility/mDDuElement.js

```diff
@@ -0,0 +1,6 @@
+export default function createDurationElement(duration, element) {
+  const taskCompleteDuration = document.createElement("div");
+  taskCompleteDuration.classList.add("duration");
+  taskCompleteDuration.textContent = `Completed in ${duration} days`;
+  element.appendChild(taskCompleteDuration);
+}
```

Comment:
The name `createDurationElement` suggests that it's a factory function for duration element. But internally it's mutating the element by using `appendChild`. 

We can simply return the `taskCompleteDuration` element and append it in the caller scope.

Reply:
done!